Module import error

Hi I've added your module and I'm running in an error importing some module. 

Just made a base nuxt 3 project, and then added your module as per instructions.

My package.json:
```
{
  "private": true,
  "scripts": {
    "build": "nuxt build",
    "dev": "nuxt dev",
    "generate": "nuxt generate",
    "preview": "nuxt preview",
    "postinstall": "nuxt prepare"
  },
  "devDependencies": {
    "nuxt": "^3.1.2"
  },
  "dependencies": {
    "@apollo/server": "^4.3.3",
    "@as-integrations/h3": "^1.1.5",
    "graphql": "^16.6.0",
    "nuxt-graphql-server": "^1.1.6"
  }
}
```

During prepare and runtime I run into the following error after adding the module in nuxt.config.ts:

` ERROR  Error while requiring module nuxt-graphql-server: Error [ERR_MODULE_NOT_FOUND]: Cannot find module '/home/runner/work/nuxt-graphql-server/nuxt-graphql-server/node_modules/.pnpm/jiti@1.16.0/node_modules/jiti/lib/index.js' imported from /Users/matthiaspoortvliet/CODE/test/node_modules/nuxt-graphql-server/dist/module.mjs
`

--- a/package.json
+++ b/package.json
@@ -17,7 +17,7 @@
     "dist"
   ],
   "scripts": {
-    "prepare": "nuxt-module-build --stub && nuxi prepare playground",
+    "postinstall": "nuxt-module-build --stub && nuxi prepare playground",
     "prepack": "nuxt-module-build && cp src/graphql-server.d.ts dist",
     "build": "pnpm prepack",
     "dev": "nuxi dev playground",

